packages/rest-generator/src/parse-open-api-json.ts

```diff
@@ -0,0 +1,88 @@
+import { readJsonSync } from 'fs-extra';
+import { last } from '@sap-cloud-sdk/util';
+import {
+  OpenApiOperation,
+  OpenApiPath,
+  OpenApiServiceMetadata,
+  SupportedOperation
+} from './open-api-types';
+
+export function toOpenApiModel(
```

Comment:
no doc. Also it returns a `ApiServiceMetadata` and is called `toOpenApiModel`. I would consider renaming either the method or the interface.